cardano-cli/src/Cardano/CLI/Run/Hash.hs

```diff
@@ -94,30 +94,79 @@ runHashAnchorDataCmd Cmd.HashAnchorDataCmdArgs{toHash, hashGoal} = do
     :: ExceptT FetchURLError IO BS8.ByteString -> ExceptT HashCmdError IO BS8.ByteString
   fetchURLToHashCmdError = withExceptT HashFetchURLError
 
-data SupportedSchemas = FileSchema | HttpSchema | HttpsSchema | IpfsSchema
-  deriving (Show, Eq)
-
-allSchemas :: [SupportedSchemas]
-allSchemas = [FileSchema, HttpSchema, HttpsSchema, IpfsSchema]
+-- | Specifies the treatment for the different schemes when checking anchor data from URLs.
+data SupportedSchemes = SupportedSchemes
+  { supportedSchemes :: [AnchorScheme]
```

Comment:
We are unnecessarily complicating this data definition. Adding `HttpScheme` to  `AnchorScheme` with a comment is fine.

Reply:
Addressed now, and squashed, because there was a bug in the previous commit